http.c

```diff
@@ -48,13 +50,13 @@ char **argv;
   }
   i = 1;
 
-  if (!strcasecmp(argv[i], "put")) {
+  if (!strcmp(argv[i], "put")) {
```

Comment:
well, that's not the same, there isn't a case insensitive compare on windows? (it's POSIX.1 so should be there, maybe in strings.h ?)

Reply:
IIRC, strcasecmp isn't available on Windows, but _stricmp is.

I can ifdef it in both files to keep the existing functionality.

Reply:
could be something like
https://github.com/osmcode/libosmium/issues/59
(from a random search)

Reply:
Yep, that's what I was trying to imply when I said ifdef it.